docs/application/tizen-studio/release-notes/3-1-release-notes.md

```diff
@@ -8,36 +8,36 @@
 
 ### New Features
 -   OpenJDK
-	-  Added support for OpenJDK 10.
+	-  The support for OpenJDK 10 has been added.
 
 ### Fixed Bugs
 -   Tizen IDE
 	-   Changing the profile and platform version now updates the rootstrap in project configuration.
-	-   Modified the error message, which appears while adding an incorrect privilege file to a native application.
-	-   Fixed the issues with log filtering in log tab.
+	-   The error message, which appears while adding an incorrect privilege file to a native application has been modified.
+	-   The issues with log filtering in log tab have been fixed.
 -   Certificate manager
-	-   Fixed the failure in generating a new certificate profile after 01 January 2019.
-	-   Fixed the issue in network proxy settings. Now, the network proxy setting change is applied for all network accesses.
-	-   Made the unnecessary input fields optional in UI while creating IoT certificates. 
+	-   The failure in generating a new certificate profile after 01 January 2019 has been fixed.
+	-   The issue in network proxy settings has been fixed. Now, the network proxy setting change is applied for all network accesses.
+	-   The unnecessary input fields have been changed to optional in UI while creating IoT certificates. 
 -   Device Manager
-	-   Fixed the issues with log filtering in log tab.
+	-   The issues with log filtering in log tab have been fixed.
 -   TizenRT
```

Comment:
Tizen RT? TizenRT?

Reply:
@mijin85cho, it is TizenRT

Reply:
@sabeena21 If so, would you please fix the "Tizen RT" in line 23 from 3-0-release-notes.md?

Reply:
@mijin85cho, thank you for pointing it out.
I have changed "Tizen RT" to "TizenRT"